dev/sg/checks.go

```diff
@@ -36,6 +36,7 @@ var checks = map[string]check.CheckFunc{
 	"yarn":                  check.Combine(check.InPath("yarn"), checkYarnVersion("~> 1.22.4")),
 	"go":                    check.Combine(check.InPath("go"), checkGoVersion("1.17.5")),
 	"node":                  check.Combine(check.InPath("node"), check.CommandOutputContains(`node -e "console.log(\"foobar\")"`, "foobar")),
+	"rust":                  check.Combine(check.InPath("cargo"), check.CommandOutputContains(`cargo version`, "1.60.0")),
```

Comment:
That's more recent than what we have in `.tools-version`
```suggestion
	"rust":                  check.Combine(check.InPath("cargo"), check.CommandOutputContains(`cargo version`, "1.58.0")),
```